src/airtable_clj/core.clj

```diff
@@ -53,7 +51,7 @@
                               :multi-param-style :array}
                        (seq query-params) (assoc :query-params query-params))
         response (http/get url http-options)
-        _ (handle-api-error response)
+        _ (handle-api-error response select options)
```

Comment:
I'm not sure how does 
```clojure
        _ (handle-api-error response select options)
```
works. It uses an underscore, means ignores other result (?).

Reply:
Yes—it means that the result is ignored. This will `throw` an error if there was a response error.